drivers/windows/thread_windows.cpp

```diff
@@ -78,11 +78,29 @@ void ThreadWindows::wait_to_finish_func_windows(Thread *p_thread) {
 	CloseHandle(tp->handle);
 	//`memdelete(tp);
 }
+typedef HRESULT(WINAPI *PSETTHREADDESCRIPTION)(HANDLE hThread, PCWSTR lpThreadDescription);
```

Comment:
Add a cosmetic separation line before this.

Reply:
I have done this.